lib/tapioca/dsl/compilers/active_model_secure_password.rb

```diff
@@ -78,7 +78,15 @@ def decorate
 
           root.create_path(constant) do |klass|
             methods.each do |method|
-              create_method_from_def(klass, constant.instance_method(method))
+              if method.start_with?("authenticate")
```

Comment:
Do you mind changing this check to:
```suggestion
              if method.start_with?("authenticate_") || method.to_s == "authenticate"
```
which I feel is slightly safer since it will match a narrower set.